lang/test/org/partiql/lang/syntax/SqlParserDateTimeTests.kt

```diff
@@ -25,6 +34,567 @@ class SqlParserDateTimeTests : SqlParserTestBase() {
+            tokenValue = ion.newString("34:59")
+        ),
+        // This is a valid time string in PostgreSQL
+        createErrorCaseForTime(
+            source = "TIME '59.12345'",
+            line = 1L,
+            col = 6L,
+            errorCode = ErrorCode.PARSE_INVALID_TIME_STRING,
+            tokenType = TokenType.LITERAL,
+            tokenValue = ion.newString("59.12345")
+        ),
+        // This is a valid time string in PostgreSQL
+        createErrorCaseForTime(
+            source = "TIME '1:30:38'",
+            line = 1L,
+            col = 6L,
+            errorCode = ErrorCode.PARSE_INVALID_TIME_STRING,
+            tokenType = TokenType.LITERAL,
+            tokenValue = ion.newString("1:30:38")
+        ),
+        // This is a valid time string in PostgreSQL
+        createErrorCaseForTime(
+            source = "TIME '1:30:38'",
+            line = 1L,
+            col = 6L,
+            errorCode = ErrorCode.PARSE_INVALID_TIME_STRING,
+            tokenType = TokenType.LITERAL,
+            tokenValue = ion.newString("1:30:38")
+        ),
+        createErrorCaseForTime(
+            source = "TIME '12:59:61.0000'",
+            line = 1L,
+            col = 6L,
+            errorCode = ErrorCode.PARSE_INVALID_TIME_STRING,
+            tokenType = TokenType.LITERAL,
+            tokenValue = ion.newString("12:59:61.0000")
+        ),
+        createErrorCaseForTime(
+            source = "TIME '12.123:45.123:54.123'",
+            line = 1L,
+            col = 6L,
+            errorCode = ErrorCode.PARSE_INVALID_TIME_STRING,
+            tokenType = TokenType.LITERAL,
+            tokenValue = ion.newString("12.123:45.123:54.123")
+        ),
+        createErrorCaseForTime(
+            source = "TIME '-19:45:13'",
+            line = 1L,
+            col = 6L,
+            errorCode = ErrorCode.PARSE_INVALID_TIME_STRING,
+            tokenType = TokenType.LITERAL,
+            tokenValue = ion.newString("-19:45:13")
+        ),
+        createErrorCaseForTime(
+            source = "TIME '24:00:00'",
+            line = 1L,
+            col = 6L,
+            errorCode = ErrorCode.PARSE_INVALID_TIME_STRING,
+            tokenType = TokenType.LITERAL,
+            tokenValue = ion.newString("24:00:00")
+        ),
+        createErrorCaseForTime(
+            source = "TIME '23:59:59.99999 05:30'",
+            line = 1L,
+            col = 6L,
+            errorCode = ErrorCode.PARSE_INVALID_TIME_STRING,
+            tokenType = TokenType.LITERAL,
+            tokenValue = ion.newString("23:59:59.99999 05:30")
+        ),
+        createErrorCaseForTime(
+            source = "TIME '23:59:59+05:30.00'",
+            line = 1L,
+            col = 6L,
+            errorCode = ErrorCode.PARSE_INVALID_TIME_STRING,
+            tokenType = TokenType.LITERAL,
+            tokenValue = ion.newString("23:59:59+05:30.00")
+        ),
+        // TODO: Investing why the build failed in GH actions for these two tests.
```

Comment:
typo: investigate* 
This is mostly because `LocalTime.parse` behaves differently for JDK11.